src/registrar/templates/domain_dns_record.html

```diff
@@ -0,0 +1,131 @@
+{% extends "domain_base.html" %}
+{% load static field_helpers url_helpers %}
+
+{% block title %}Prototype DNS | {{ domain.name }} | {% endblock %}
+
+{% block domain_content %}
+
+    {% include "includes/form_errors.html" with form=form %}
+
+    <h1>Add DNS records</h1>
+
+    <p>
+    This is a prototype that demonstrates adding an 'A' record to a zone.
+    Do note that this just adds records, but does not update or delete existing ones.
+    </p>
+
+    <div x-data="{ show_form: false }">
+        <button
+        class="usa-button"
+        type="button"
+        x-on:click="show_form = true"
+        >
+            Add record
+        </button>
+        <form 
+            class="usa-form usa-form--large"
+            method="post"
+            novalidate id="form-container"
+            x-cloak
+            x-show="show_form"
+        >
+            {% csrf_token %}
+            {% input_with_errors form.name %}
+            {% input_with_errors form.content %}
+            {% input_with_errors form.ttl %}
+            <button
+                    type="submit"
+                    class="usa-button"
+                >
+                    Submit record
+            </button>
+            <button
+                type="button"
+                class="usa-button usa-button--outline"
+                x-on:click="show_form = false"
+            >
+                Cancel
+            </button>
+        </form>
+        
+    </div>
+
+    {% if dns_records %}
```

Comment:
What would the implications be of moving the table into it's own template? Would that complicate adding a row after submission using htmx?